client/my-sites/upgrades/domain-management/name-servers/dns-template-button-group.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+
+class DnsTemplateButtonGroup extends Component {
```

Comment:
This is perfect! @hambai and I were actually looking for a way to add some sort of "selected" indicator to ButtonGroup, but we somehow missed seeing SegementedControl.